easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.1.1-33-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,44 @@
+# Update: Ehsan Moravveji (VSC - KU Leuven)
+
+easyblock = 'ConfigureMake'
+
+name = 'ImageMagick'
+version = '7.1.1-33'
```

Comment:
A slightly newer version of ImageMagick for GCCcore 13.2.0 got merged yesterday:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/i/ImageMagick/ImageMagick-7.1.1-34-GCCcore-13.2.0.eb
So you can remove this one from your PR and stick to that version (I don't see your version being used in this PR anyway?).

Reply:
It is used by the bundle. But yes needs updating.

And a new R EasyConfig sneeked in

Reply:
- ImageMagick is adapted in 7f4948622988635b10ce702a4b9e87cd61945b75
- The sneaky R is taken out in b39bd1b24959e948d1debee0a6a4845b301f1698

Reply:
Ah, I overlooked that. Guess I was confused by the R easyconfig (and the bundle one not being expanded due to the large diff).